### ISSUE TITLE
Unescaped template parameter in [exec.with.awaitable.senders]

### ISSUE THREAD

**Issue by lewissbaker** [cplusplus/sender-receiver#272]
In P2300R10 [exec.with.awaitable.senders] p2 there is the function signature:
```
template<class OtherPromise>
  requires (!same_as<OtherPromise, void>)
void set_continuation(coroutine_handle h) noexcept;
```
I think the parameter type is supposed to be `coroutine_handle<OtherPromise>` (there is an unescaped less-than character in the markup)

**Comment by ericniebler** [comment]
fixed in [12fde4a]

**Comment by lewissbaker** [comment]
This fix will need to be applied to the working draft now. Can we leave this open for now?
